src/lib/Bond.ts

```diff
@@ -111,19 +94,6 @@ export abstract class Bond {
     const bondAddress = this.getAddressForReserve(NetworkId) || "";
     return new ethers.Contract(bondAddress, this.reserveContract, provider) as PairContract;
   }
-
-  // TODO (appleseed): improve this logic
-  async getBondReservePrice(NetworkId: NetworkId, provider: StaticJsonRpcProvider | JsonRpcSigner) {
-    let marketPrice: number;
-    if (this.isLP) {
-      const pairContract = this.getContractForReserve(NetworkId, provider);
-      const reserves = await pairContract.getReserves();
-      marketPrice = Number(reserves[1].toString()) / Number(reserves[0].toString()) / 10 ** 9;
-    } else {
-      marketPrice = await getTokenPrice("convex-finance");
-    }
-    return marketPrice;
-  }
 }
 
```

Comment:
Can we remove this file entirely?

Reply:
It could potentially be slimmed down. We still use it for `AllBonds.ts`, which is still used for `ohm_dai` pricing. There may be more.

Reply:
wait nevermind... I'm looking closer now

Reply:
ok slimmed it way down to bear bones. once we remove accountSlice (need migration to complete) then this can be removed.